### ISSUE TITLE
Rasberry Pi - how to install?

### ISSUE THREAD

**Issue by cloudjumper2000** [ContainX/docker-volume-netshare#187]
Downloaded and tried to install using both of these with no such luck. Whats the process to install on ARM?
docker-volume-netshare_0.36_linux_arm-bin
docker-volume-netshare_0.36_linux_arm.tar.gz

Thx!

**Comment by g0lg0th7** [comment]
RTFM

**Comment by cloudjumper2000** [comment]
While it probably feels really good to write RTFM; 

I read through both the readme here:
https://github.com/ContainX/docker-volume-netshare
And the install here: 
http://netshare.containx.io/docs/getting-started

I didnt find any mention on how to install from .bin file nor does it mention ARM installation that I found.

**Comment by g0lg0th7** [comment]
1) get https://github.com/ContainX/docker-volume-netshare/releases/download/v0.36/docker-volume-netshare_0.36_armhf.deb
2) dpkg -i docker-volume-netshare_0.35_<arch>.deb
3) docker-volume-netshare -h --> docker-volume-netshare nfs
4) read https://sysadmins.co.za/docker-swarm-persistent-storage-with-nfs/ --> thanks Ruan
5) implement at your context
From my understanding.

**Comment by g0lg0th7** [comment]
Sorry, bin archive can be used as is :
docker-volume-netshare_0.36_linux_arm-bin nfs as per step 3)

**Comment by rockaut** [comment]
Yeah ... sadly no. This doesn't work on an RPi4 for example as its arm64. Maybe @cloudjumper2000 was refering to that.
